apps/prairielearn/src/ee/lib/ai-grading/ai-grading-stats.ts

```diff
@@ -0,0 +1,129 @@
+import assert from 'node:assert';
+
+import { z } from 'zod';
+
+import { loadSqlEquiv, queryOptionalRow, queryRows } from '@prairielearn/postgres';
+import { DateFromISOString } from '@prairielearn/zod';
+
+import {
+  type AssessmentQuestion,
+  IdSchema,
+  type RubricItem,
+  RubricItemSchema,
+} from '../../../lib/db-types.js';
+
+const sql = loadSqlEquiv(import.meta.url);
+const GradingJobInfoSchema = z.object({
+  grading_job_id: IdSchema,
+  graded_at: DateFromISOString.nullable(),
+  grading_method: z.enum(['Manual', 'AI']),
+  manual_points: z.number().nullable(),
+  manual_rubric_grading_id: IdSchema.nullable(),
+  grader_name: z.string(),
+  rubric_items: z.array(RubricItemSchema),
+});
+type GradingJobInfo = z.infer<typeof GradingJobInfoSchema>;
+
+export interface AIGradingStats {
+  last_human_grader: string | null;
+  ai_grading_status: 'Graded' | 'Latest' | 'Outdated' | 'None';
```

Comment:
Nit: should we name these `LatestRubric` and `OutdatedRubric` instead to make it clear that whether or not something it "latest" is tied to the rubric somehow?